dev/com.ibm.ws.security.wim.adapter.ldap/src/com/ibm/ws/security/wim/adapter/ldap/LdapAdapter.java

```diff
@@ -3505,6 +3505,13 @@ else if (SchemaConstants.DO_GROUPS.equals(propertyName)) {
             } else if (SchemaConstants.DATA_TYPE_LANG_TYPE.equals(dataType)) {
                 LangType langDO = (LangType) value;
                 ldapValue = langDO.getValue();
+            } else if (SchemaConstants.DATA_TYPE_BOOLEAN.equals(dataType)) {
+                /*
+                 * According to RFC 451 SEctrion 3.3.3, The LDAP-specific encoding of a value of this syntax is defined by the following ABNF: Boolean = "TRUE" / "FALSE"
```

Comment:
```suggestion
Some servers (OpenLDAP, Micorosoft Active Directory, among others) won't accept lower-case Boolean values according to RFC 4517, section 3.3.3, where the ABNF is:

    Boolean = "TRUE" / "FALSE"

ABNF literal strings are defined as case-insensitive. Always send the upper-case value instead of the lower-case value as it is most likely that LDAP servers are using the upper-case values to validate the syntax if they are applying any case-sensitivity to syntax validation.
```